elements/rh-footer/test/rh-footer.spec.ts

```diff
@@ -2,15 +2,26 @@ import { html } from 'lit';
 import { fixture, expect, nextFrame, aTimeout } from '@open-wc/testing';
 import { setViewport } from '@web/test-runner-commands';
 import { RhFooter } from '../RhFooter.js';
+import { RhGlobalFooter } from '../rh-global-footer.js';
```

Comment:
I'm adding some basic tests for the global footer.  This element would benefit greatly with some visual regression snapshots. 😀